connectors/aws/configurations/lang_en.json

```diff
@@ -29,12 +29,12 @@
                 "description": "Account ID of Amazon Web Service for authentication"
             },
             "aws_access_key_id": {
-                "label": "Client ID",
-                "description": "Client ID of Amazon Web Service for authentication"
+                "label": "AWS access key ID",
+                "description": "AWS Access Key ID is required for both AWS key-based and role-based authentication"
             },
             "aws_secret_access_key": {
-                "label": "Client secret",
```

Comment:
Not sure why are these fields called Client ID and secret! can you please verify the reason?

Reply:
I'm not sure why they originally had those names. However it using boto3.

CAR code
`    def getClient(self, service_name):
        return boto3.client(service_name, aws_access_key_id=self.aws_access_key_id,
                                   aws_secret_access_key=self.aws_secret_access_key,
                                   region_name=self.region_name)

    def getResource(self, service_name):
        return boto3.resource(service_name, aws_access_key_id=self.aws_access_key_id,
                                   aws_secret_access_key=self.aws_secret_access_key,
                                   region_name=self.region_name)`

Which has documentation here https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html 

that specifies that it needs AWS access/secret keys.

Reply:
@DerekRushton I don't think `aws_access_key_id ` and `aws_secret_access_key `.